eiffel-vocabulary/EiffelServiceAllocatedEvent.md

```diff
@@ -19,7 +19,36 @@
 The EiffelServiceAllocatedEvent is to declare an instance has been allocated to a User.( e.g global scheduler or resource booking tool).
 
 ## Data Members
- 
+### data.serviceinstance
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The name of the Service instance that was allocated
+
+### data.owner
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The user that owns the Service instance
```

Comment:
Is it the execution user that runs the instance, or the responsible human behind this?

Reply:
The "user" is a person or a machine identity.

Proposal to change Description to :
The entity that has been assigned/responsible for the Service.